src/App.js

```diff
@@ -1,25 +1,49 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+
+import "./App.css";
+
+const App = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+  const [changeBackgroundColor, setChangeBackgroundColor] = useState(true);
 
-function App() {
```

Comment:
If file contains any JSX then it's type should be .jsx instead of .js (change src/App.js -> src/App.jsx)